src/modules/checks/suites/consensus_node.py

```diff
@@ -4,20 +4,24 @@
 
 def check_validators_provided(web3: Web3, blockstamp):
     """Check that consensus-client able to provide validators"""
-    result = web3.cc.get_validators_no_cache(blockstamp)
-    assert len(result) > 0, "consensus-client provide no validators"
+    assert web3.cc.get_validators_no_cache(blockstamp), "consensus-client provide no validators"
 
 
 def check_block_details_provided(web3: Web3, blockstamp):
     """Check that consensus-client able to provide block details"""
-    web3.cc.get_block_details(blockstamp.slot_number)
+    assert web3.cc.get_block_details(blockstamp.slot_number), "consensus-client provide no block details"
 
 
 def check_block_root_provided(web3: Web3, blockstamp):
     """Check that consensus-client able to provide block root"""
-    web3.cc.get_block_root(blockstamp.slot_number)
+    assert web3.cc.get_block_root(blockstamp.slot_number), "consensus-client provide no block root"
 
 
 def check_block_header_provided(web3: Web3, blockstamp):
     """Check that consensus-client able to provide block header"""
-    web3.cc.get_block_header(blockstamp.slot_number)
+    assert web3.cc.get_block_header(blockstamp.slot_number), "consensus-client provide no block header"
+
+
+def check_block_roots_from_state_provided(web3: Web3, blockstamp):
+    """Check that consensus-client able to provide block roots from state"""
+    assert web3.cc.get_state_block_roots(blockstamp.slot_number), "consensus-client provide no block roots from state"
```

Comment:
What blockstamp is being used here?

Reply:
either `finalized` or `finalized - frame size`

Reply:
Does it make sense? I think we should to step back for CL tests to make sure we have historical state?

Reply:
frame_size from what?

Reply:
> Does it make sense? I think we should to step back for CL tests to make sure we have historical state?

I changed nothing here. The test will be run with finalized slot and with slot with -270 epochs back. see `blockstamp_frame_ago` fixture